system/services/formbuilder/FormBuilderService.cfc

```diff
@@ -959,6 +959,46 @@ component {
 		}
 	}
 
+	public string function cloneForm(
+		  required string basedOnFormId
+		, required string name
+		, required string description
+	) {
+		var originalFormData = getForm( id=arguments.basedOnFormId );
+		var cloneFormData = structNew();
```

Comment:
Here, I think this is more accurate:

```
var cloneFormData = { name=arguments.name, description=arguments.description };
```

Then further down, when you loop over the `originalFormData.columnList`, you can ignore these fields and remove the logic on line 972-3.